### DISCUSSION TITLE
Mouse position in GUI

### DISCUSSION THREAD

**Discussion by norwichrunner** [BlitterStudio/amiberry#953] [Q&A]
Hiya, I've been having a problem with the mouse in GUI (retropie, v5.0).  Vertically it seems to be off by about an inch, so I have to click above or below things.  Its fine in games - do you have any idea what might be causing this please?

**Comment by Retro1968** [comment]
I had mouse issues after a firmwareupdate. https://github.com/midwan/amiberry/issues/940

**Comment by midwan** [comment]
@norwichrunner Do you have Overscan enabled perhaps? That's a common reason for the behavior you're describing.
